### PR TITLE
Refactor/units list

### PR DESCRIPTION
**Pull request by wwills2**
this PR adds the units page, units list table, and units api. 

misc: added types for flowbite proxy components

problems fixed: 
- skeleton table data missing unique key
- missing projects table translations
- non-translatable pure text
- ts types for datatable, projects list table, and appLogo
- removed direct flowbite imports

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     
     <link rel="icon" type="image/svg+xml" href="./logo.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Sprout UI - Chia dApp ProjectsList</title>
+    <title>Sprout UI - Chia dApp UnitsList</title>
```
Comment (MichaelTaylor3D) [inline index.html:8]: Not sure what this is but it should saw "Climate Action Data Trust"

File: src/renderer/api/cadt/v1/units/units.api.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          params.search = search.replace(/[^a-zA-Z0-9 _.-]+/, '');
+        }
+
+        return {
+          url: `${host}/v1/units`,
+          params, // Use the constructed params object
+          method: 'GET',
+        };
+      },
+      providesTags: [unitsTag],
```
Comment (MichaelTaylor3D) [inline src/renderer/api/cadt/v1/units/units.api.ts:38]: we want projects and tags to include the orgUid so we can invalidate just the orgUid

Comment (MichaelTaylor3D) [reply src/renderer/api/cadt/v1/units/units.api.ts:38]: `providesTags: (result, error, {orgUid} ) => [{ type: 'units', id: orgUid }],`

**Review by MichaelTaylor3D — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (wwills2) [reply index.html]: corrected.this was a hold over from sprout-ui

**Pull request merged**